src/index.html

```diff
@@ -1,17 +1,161 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
There should be no blank lines between parent and child blocks
```suggestion
<html lang="en">
  <head>
```